### PR TITLE
release: 6.5.0

### PR DESCRIPTION
**Pull request by katspaugh**
6.5.0
------------------
- Regions plugin:
  - Improved delta calculation (resize end) (#2641)
- Fix iphone silent switch webaudio mute (#2667)
- Respect mute state when changing volume during mute (#2502)

### CODE REVIEW

File: CHANGES.md [hunk 1]
```diff
@@ -1,12 +1,17 @@
...
+6.5.0 (11.03.2023)
 ------------------
 - Regions plugin:
   - Improved delta calculation (resize end) (#2641)
 - Fix iphone silent switch webaudio mute (#2667)
 - Respect mute state when changing volume during mute (#2502)
+- Proxy HTMLMediaElement's 'waiting' event through MediaElement backend. (Proxy HTMLMediaElement's 'waiting' event through MediaElement backend. #2691)
+- Chores:
+  - Docs: update the issue template and wavesurfer links (Docs: update the issue template and wavesurfer links #2671)
+  - Create FUNDING.yml (Create FUNDING.yml #2668)
```
Comment (thijstriemstra) [inline CHANGES.md:13]: this is the type of useless changelog messages i don't like when automating things; please give it a meaningful description.

File: CHANGES.md [hunk 2]
```diff
@@ -1,12 +1,17 @@
...
 =======================
 
-6.5.0 (unreleased)
+6.5.0 (11.03.2023)
 ------------------
 - Regions plugin:
   - Improved delta calculation (resize end) (#2641)
 - Fix iphone silent switch webaudio mute (#2667)
 - Respect mute state when changing volume during mute (#2502)
+- Proxy HTMLMediaElement's 'waiting' event through MediaElement backend. (Proxy HTMLMediaElement's 'waiting' event through MediaElement backend. #2691)
```
Comment (thijstriemstra) [inline CHANGES.md:10]: this changelog entry has everything listed twice, why?

File: CHANGES.md [hunk 3]
```diff
@@ -1,12 +1,17 @@
...
+- Chores:
+  - Docs: update the issue template and wavesurfer links (Docs: update the issue template and wavesurfer links #2671)
```
Comment (thijstriemstra) [inline CHANGES.md:12]: don't list things twice

**Review by thijstriemstra — CHANGES_REQUESTED** [review]

File: CHANGES.md (hunk 2 above)
Comment (katspaugh) [reply CHANGES.md]: Good catch, fixed!

File: CHANGES.md [hunk 4]
```diff
@@ -1,12 +1,17 @@
...
 ------------------
 - Regions plugin:
   - Improved delta calculation (resize end) (#2641)
 - Fix iphone silent switch webaudio mute (#2667)
 - Respect mute state when changing volume during mute (#2502)
+- Proxy HTMLMediaElement's 'waiting' event through MediaElement backend. (#2691)
+- Chores:
+  - Docs: update the issue template and wavesurfer links (#2671)
+  - Add FUNDING.yml to enable GitHub Sponsors (#2668)
+  - update dev dependencies
```
Comment (thijstriemstra) [inline CHANGES.md:15]: no need to add the update dev dependencies line

**Review by thijstriemstra — CHANGES_REQUESTED** [review]

File: CHANGES.md (hunk 4 above)
Comment (katspaugh) [reply CHANGES.md:15]: Why? It's good to know that the tooling is updated if you're a contributor.

Comment (thijstriemstra) [reply CHANGES.md:15]: you can assume it's always updated (usually several times during dev cycle of a specific version), 100% of the time, that's why I would exclude it. If you don't want to exclude it, please start sentence with capital and use full sentence, e.g. `Update development dependencies`

**Review by thijstriemstra — APPROVED** [review]
thanks!

File: CHANGES.md (hunk 4 above)
Comment (katspaugh) [reply CHANGES.md:15]: I've changed it as requested, and appreciate the attention to detail, but I think we should work out a less pedantic/more pragmatic approach to pull request reviews in the future.

The reviewer is there to help and support the contributor with expert knowledge of the codebase and an encouraging word. If we make it harder to contribute by requesting several rounds of tiny changes, pull requests will be more likely to sit stale for months.

**Comment by thijstriemstra** [comment]
@katspaugh unfortunately the publish process failed: https://github.com/wavesurfer-js/wavesurfer.js/actions/runs/4392444883/jobs/7692214471#step:7:14

**Comment by katspaugh** [comment]
On it, the GitHub API key needs to be updated due to the migration to the new org.

**Comment by thijstriemstra** [comment]
Gotcha. After that you should be able to re-run that specific github actions job to publish it again.

**Pull request merged**